x-pack/plugins/security_solution/public/timelines/store/timeline/epic.ts

```diff
@@ -414,7 +411,7 @@ export const convertTimelineAsInput = (
                   ...(esFilters.isRangeFilter(basicFilter) && basicFilter.range != null
                     ? { range: convertToString(basicFilter.range) }
                     : { range: null }),
-                  ...(esFilters.isRangeFilter(basicFilter) &&
+                  ...(isScriptedRangeFilter(basicFilter) &&
```

Comment:
Is this the same as before which was a `rangeFilter` vs a scripted range filter?

Right now this looks incorrect and you wanted `isRangeFilter` here instead?

Reply:
We are trying to get the `script` attribute which only exists on the ScriptedRange filter.
Before this PR, the `RangeFilter` implicitly had both, but now I've split them out into two types.
So I think this is the correct form. What do you think?

Reply:
I think I see it since above this we have a `isRangeFilter`